CHANGELOG.md

```diff
@@ -28,6 +28,8 @@
   - implement new UI design for customer detail data overlay
 - **Company Subscription Management**
   - extended the subscription detail page for customer to include more app-related information [#1051](https://github.com/eclipse-tractusx/portal-frontend/pull/1051)
+- **Clearinghouse Self-Description**
+  - Admin UI for Managing SD Document Retriggering
```

Comment:
please make sure to add the comment in the changelog under the [`Unreleased` section](https://github.com/eclipse-tractusx/portal-frontend/blob/main/CHANGELOG.md#unreleased) and please also add the link to the pull request.

Reply:
Changes Done